root/static/scripts/release-editor/duplicates.js

```diff
@@ -170,16 +170,16 @@ function formatReleaseData(release) {
 
 
 function combinedMediumFormatName(mediums) {
-    var formats = pluck(_(mediums), "format");
-    var formatCounts = formats.countBy(_.identity);
+    const getFormatOrUnknown = medium => medium.format || '(unknown)';
```

Comment:
`'(unknown)'` is in the mb_server domain, so may not work being passed to `lp_attributes` below. Perhaps just fallback to `''` here and do something like `format ?  lp_attributes(format, 'medium_format') : l('(unknown)')` below?
```suggestion
    const getFormatOrUnknown = medium => medium.format || '';
```

Reply:
See the commit message: 

> While this isn't currently being translated, it will eventually need to
use "medium_format" context for consistency with the rest. This is
different from the current usage for '(unknown)', but IMO preferable,
since a medium format and, say, a release status might need different
forms of the word "unknown" in different languages.

Reply:
Sorry! I should've went through the commits first.

Gotcha, but are you suggesting we include it in the attributes domain somehow, or instead wouldn't we want to use `lp('(unknown)', 'medium_format')` at least? Since it also can't be extracted this way either, unless we pass the string directly to `lp` or `lp_attributes`.

Reply:
Correction, only if we pass it directly to `lp` at present (since we don't extract strings from domain-specific calls).

Reply:
Left it as l() for now but added MBS-10607 for the issue with translating all 'unknown' as the same.